### PR TITLE
docs - remove gptransfer from docs

### PR DESCRIPTION
**Pull request by mkiyama**
--removed gptransfer topics, references to gptransfer, and images.
--also updated text in gpcopy-migrate as rough update for 6.0

PR for removing gptransfer https://github.com/greenplum-db/gpdb/pull/6794

Link to HTML format of updated docs on a temporary GPDB review site.

https://docs-msk-gpdb6-gptrans-dev.cfapps.io/600/admin_guide/managing/gpcopy-migrate.html
https://docs-msk-gpdb6-gptrans-dev.cfapps.io/600/admin_guide/managing/partII.html

https://docs-msk-gpdb6-gptrans-dev.cfapps.io/600/best_practices/summary.html

https://docs-msk-gpdb6-gptrans-dev.cfapps.io/600/install_guide/apx_mgmt_utils.html
https://docs-msk-gpdb6-gptrans-dev.cfapps.io/600/install_guide/prep_os_install_gpdb.html

https://docs-msk-gpdb6-gptrans-dev.cfapps.io/600/security-guide/topics/ports_and_protocols.html

https://docs-msk-gpdb6-gptrans-dev.cfapps.io/600/utility_guide/dblink.html
https://docs-msk-gpdb6-gptrans-dev.cfapps.io/600/utility_guide/admin_utilities/util_ref.html

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/managing/gpcopy-migrate.xml [hunk 1]
```diff
@@ -19,13 +22,13 @@
...
-        <codeph>gptransfer</codeph> utility. <codeph>gpcopy</codeph> provides several improvements
-      as compared to <codeph>gptransfer</codeph>:<ul id="ul_ibs_vsp_zdb">
-        <li><codeph>gpcopy</codeph> performs significantly faster than <codeph>gptransfer</codeph>
-          when copying large amounts of data.</li>
-        <li><codeph>gpcopy</codeph> provides more detailed reporting and summary information about
-          all aspects of the copy operation.</li>
+    <p><codeph>gpcopy</codeph> is a command-line tool that includes these features:<ul
+        id="ul_ibs_vsp_zdb">
+        <li><codeph>gpcopy</codeph> can migrate data between Greenplum Database systems where the
+          source and destination systems are configured with of a different number of segment
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/managing/gpcopy-migrate.xml]: with of a -> with a

File: gpdb-doc/dita/admin_guide/managing/gpcopy-migrate.xml [hunk 2]
```diff
@@ -54,7 +57,7 @@
         cluster to a Greenplum Database 5.9 or later cluster. However Greenplum Database 4.3.26 and
         later do not include the actual <codeph>gpcopy</codeph> utility. You must manually copy the
           <codeph>gpcopy</codeph> utility from your version 5.9 or later cluster into the older
-        version cluster to migrate data.. For
+        version cluster to migrate data. For
         example:<codeblock>$ cp /usr/local/greenplum-db-5.8.0/bin/gpcopy /usr/local/greenplum-db-4.3.26.0/bin/</codeblock></p>
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/managing/gpcopy-migrate.xml]: Should update the example to use greenplum-db-5.9.0 to match text

File: gpdb-doc/dita/utility_guide/utility_guide.ditamap [hunk 3]
```diff
@@ -45,7 +50,6 @@
             <topicref href="admin_utilities/gpstate.xml"/>
             <topicref href="admin_utilities/gpstop.xml"/>
             <topicref href="admin_utilities/gpsys.xml"/>
-            <topicref href="admin_utilities/gptransfer.xml"/>
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/utility_guide.ditamap]: Should we replace this with a gpcopy link?

Comment (dyozie) [reply gpdb-doc/dita/utility_guide/utility_guide.ditamap]: Ah - never mind, that explains why it's not in the github diff.

**Pull request merged**